metadata.rb

```diff
@@ -1,15 +1,16 @@
+# frozen_string_literal: true
+
 name                'duosecurity'
 maintainer          'Matt Kulka'
 maintainer_email    'matt@lqx.net'
 license             'MIT'
 description         'Installs/Configures duosecurity two-factor system authentication on Ubuntu/Debian'
-long_description    IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version             '2.1.1'
+version             '2.2.0'
```

Comment:
3.0.0 is fine due to removing Chef compatibility with older versions.

Reply:
Good call.  Updated.